### ISSUE TITLE
About MaskRCNN Instance Segmentation Results on Cityscapes test set

### ISSUE THREAD

**Issue by hsfzxjy** [facebookresearch/detectron2#641]
The Instance Segmentation performance of MaskRCNN on Cityscapes is lower than MaskRCNN results on Cityscapes leaderboard.

## Instructions To Reproduce the Issue:

Simply run the command below to do instance segmentation inference on Cityscapes test set.

```bash
python tools/train_net.py --config-file configs/Cityscapes/mask_rcnn_R_50_FPN.yaml --eval-only --num-gpus 4 MODEL.WEIGHTS model_final_af9cf5.pkl DATASETS.TEST '("cityscapes_fine_instance_seg_test",)'
```

and then I submit the inference results to the Cityscapes official site.

## Expected behavior:

![image](https://user-images.githubusercontent.com/4702188/71877035-2b86cf80-3163-11ea-8f94-eccd3a180c1c.png)
The screenshot above is the performance of detectron2 MaskRCNN.

![image](https://user-images.githubusercontent.com/4702188/71877100-5a04aa80-3163-11ea-86da-26af3bf2b147.png)
The screenshot above is the performance of MaskRCNN on the leaderboard.

We can see that detectron2 MaskRCNN gets lower performance on AP50%, AP100m, AP50m.

I am wondering if the training settings of MaskRCNN on Cityscapes Instance Segmentation Task is equivilant to that of original MaskRCNN. Or how can I achieve the leaderboard performance using detectron2.

It will be grateful if you could share some advice!

**Comment by ppwwyyxx** [comment]
> I am wondering if the training settings of MaskRCNN on Cityscapes Instance Segmentation Task is equivilant to that of original MaskRCNN.

They are not equivalent, therefore what you observed is expected.

The paper has  "To do this, we initialize the corresponding 7 categories in Cityscapes from a
pre-trained COCO Mask R-CNN model (rider being randomly initialized)" which should help a lot. And there might be other differences I'm not aware of.

**Comment by youngwanLEE** [comment]
@hsfzxjy 
I followed your command but don't figure out where the result file was saved.

Would you let me know where the result file is saved ?
